### ISSUE TITLE
In Call UI Removed

### ISSUE THREAD

**Issue by ddubnyk** [JBNCK/One-UI-Experience#1]
Phone: S22
Android 13

I just discovered today that on the AOSP version, (and I'd suspect the Google preset), that calls do not show. I never actually got it working, however I might try play around with it still now that I know the required app. The phone calls would happen, but no UI to interact with, and it only works through earpiece. As well, this makes it impossible to end a call, without rebooting the phone. 

After restoring the Samsung Dialer, I also had to restore the "incallui". Only then could I see the UI. Also of note, I had to restore the "com.samsung.SMT" app in order to access the settings in the Samsung dialer. It just seems to crash without SMT.

**Comment by JBNCK** [comment]
Yes I also noticed this issue after upgrading to Android 13. I will update the script to Android 13 soon since I've also seen some complaints on XDA

**Comment by rudra-code-creator** [comment]
Hi is this a Python rewrite of One-UI-Experience-Archive
